app/src/main/java/org/mozilla/focus/browser/BrowserFragment.kt

```diff
@@ -162,6 +164,19 @@ class BrowserFragment : IWebViewLifecycleFragment() {
         Unit
     }
 
+    private fun shouldShowPocketOnboarding(): Boolean {
+        if (!PreferenceManager.getDefaultSharedPreferences(context).getBoolean(
+                        POCKET_ONBOARD_SHOWN_PREF, false)) {
+            PreferenceManager.getDefaultSharedPreferences(context)
+                    .edit()
+                    .putBoolean(POCKET_ONBOARD_SHOWN_PREF, true)
+                    .apply()
+            return true
+        }
+        return false
+    }
+
+
```

Comment:
\[ktlint\] WARNING: \[no\-consecutive\-blank\-lines\] Needless blank line\(s\) in column 1